### PR TITLE
Resolve RGCA #89 "handleAuthClick not returning any Promise"

### PR DESCRIPTION
**Pull request by oscarcusin**
Resolve RGCA #89

### CODE REVIEW

File: src/ApiCalendar.ts [hunk 1]
```diff
@@ -82,19 +87,30 @@ class ApiCalendar {
 
   /**
    * Sign in Google user account
+   * @returns {any} Promise resolved if authentication is successful, rejected if unsuccessful.
    */
-  public handleAuthClick(): void {
+  public handleAuthClick(): any {
```
Comment (Kubessandra) [inline src/ApiCalendar.ts:92]: If I'm not mistaken the right return type is `Promise<void>` no ?

**Review by Kubessandra — CHANGES_REQUESTED** [review]
Thanks just a comment, lets try to avoid any if we can.

File: src/ApiCalendar.ts (hunk 1 above)
Comment (oscarcusin) [reply src/ApiCalendar.ts]: Correct, fixed.

**Comment by kingdcreations** [comment]
Some news on this ?

**Comment by nhcarter123** [comment]
Looking forward to this feature.

**Comment by Kubessandra** [comment]
Reviewing it and merging it

**Comment by Kubessandra** [comment]
v2.3.0

**Pull request merged**